src/adsp_config.c

```diff
@@ -1265,6 +1265,129 @@ static int parse_signed_pkg_v2_5(const toml_table_t *toml, struct parse_ctx *pct
 	return 0;
 }
 
+static void dump_signed_pkg_v3_0(const struct signed_pkg_info_ext_v3_0 *signed_pkg)
```

Comment:
we can now call it ACE1.5, @dcpleung @lgirdwood @mwasko ?

Reply:
yeah, I think so - easier to follow e.g. use a ```_ace_v1_5``` suffix.